iOS/UI/Browse/BrowseViewController.swift

```diff
@@ -187,14 +185,27 @@ class BrowseViewController: BaseTableViewController {
         let migrateView = MigrateSourcesView()
         present(UIHostingController(rootView: SwiftUINavigationView(rootView: AnyView(migrateView))), animated: true)
     }
+
+    @objc func stopEditing() {
+        tableView.setEditing(false, animated: true)
+        self.updateNavBar()
+    }
 }
 
 // MARK: - Table View Delegate
 extension BrowseViewController {
 
+    func tableView(_ tableView: UITableView, editingStyleForRowAt indexPath: IndexPath) -> UITableViewCell.EditingStyle {
+        .none
+    }
+
+    func tableView(_ tableView: UITableView, shouldIndentWhileEditingRowAt indexPath: IndexPath) -> Bool {
+        false
+    }
+
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         if
-            sectionIdentifier(for: indexPath.section) == .installed,
```

Comment:
use a `case let` here to store the identifier then check that it's one of the two